### PR TITLE
Add codestyle check

### PR DESCRIPTION
**Pull request by knurff**

### CODE REVIEW

File: src/main/java/com/qunite/api/QuniteApiApplication.java [hunk 1]
```diff
@@ -1,7 +1,14 @@
 package com.qunite.api;
 
+import java.util.Arrays;
+
+import org.springframework.boot.CommandLineRunner;
```
Comment (Kivanval) [inline src/main/java/com/qunite/api/QuniteApiApplication.java:5]: ⚠️ **[reviewdog]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'org.springframework.boot.CommandLineRunner'

File: src/main/java/com/qunite/api/QuniteApiApplication.java [hunk 2]
```diff
@@ -10,4 +17,8 @@ public static void main(String[] args) {
         SpringApplication.run(QuniteApiApplication.class, args);
     }
 
+
+    private void DoNothing(){
```
Comment (Kivanval) [inline src/main/java/com/qunite/api/QuniteApiApplication.java:21]: ⚠️ **[reviewdog]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 4, expected level should be 2.

Comment (Kivanval) [inline src/main/java/com/qunite/api/QuniteApiApplication.java:21]: ⚠️ **[reviewdog]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name 'DoNothing' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.

File: src/main/java/com/qunite/api/QuniteApiApplication.java [hunk 3]
```diff
@@ -10,4 +17,8 @@ public static void main(String[] args) {
...
+
+    }
```
Comment (Kivanval) [inline src/main/java/com/qunite/api/QuniteApiApplication.java:23]: ⚠️ **[reviewdog]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 4, expected level should be 2.

**Review by Kivanval — APPROVED** [review]

**Review by vetterocki — APPROVED** [review]

**Pull request merged**